### ISSUE TITLE
Make above and below exclusive by default. Change open/closed to inclusive/exclusive

### ISSUE THREAD

**Issue by deontologician** [rethinkdb/horizon#118]
Suggested by @chipotle 

The words above and below sounds exclusive, so this makes sense.

The argument originally made for above being inclusive is that it's similar to how a range is usually done, making paging more natural perhaps. Since we'll have a separate paging api (#31) and above and below can be used by themselves, it makes sense to have them use the same behavior.

The protocol may need to change to `inclusive`/`exclusive` to match the client. This could be done in a backwards compatible way (only a client change) but I don't think we should worry about that yet.

**Comment by deontologician** [comment]
Anyone interested in this still? 2.0 is a good place to change it or to close this issue

**Comment by danielmewes** [comment]
I still don't like that the current names are not precisely reflecting what they do. I think commands should do what their name suggests, and not subtly differ in edge cases that are easy to forget about until something breaks.

I think we should make both of them exclusive, and add an optarg for putting them into inclusive mode.

Alternatively I'd be ok with having additional commands as you suggested:
- `.above` : exclusive lower bound
- `.below` : exclusive upper bound
- `.from` : inclusive lower bound
- `.to` : inclusive upper bound

**Comment by mlucy** [comment]
I don't think we should make them both be exclusive.  99% of the time when I'm operating on a range I want inclusive lower bound and exclusive upper bound, and that's what literally every other language's range operations do by default (including ReQL's).  Changing the behavior to an unusual and inconvenient default because it better matches the name we picked seems like a bad option to me.

I wouldn't object to picking a new name for `above` that more clearly communicates that it's inclusive, although I think `from` isn't very clear and we could probably come up with a better one.
